cl/favorites/utils.py

```diff
@@ -117,11 +124,16 @@ async def get_top_prayers() -> list[RECAPDocument]:
             "attachment_number",
             "pacer_doc_id",
             "page_count",
+            "is_free_on_pacer",
             "description",
+            "docket_entry__entry_number",
             "docket_entry__docket_id",
             "docket_entry__docket__slug",
+            "docket_entry__docket__case_name",
```

Comment:
Since `best_case_name` can also use `case_name_full` and `case_name_short`. We should add them here to avoid additional queries.

Reply:
alright!